Readme.md

```diff
@@ -12,7 +12,7 @@
 * the application requires Microsoft .NET Framework 4.6.1 which can be downloaded from [https://www.microsoft.com/download/details.aspx?id=49982](https://www.microsoft.com/download/details.aspx?id=49982).
 
 ### Download
-Download the latest release from the [release section](https://github.com/Arcitectus/Sanderling/releases).
+Download the latest release from the [release section](https://github.com/bozoweed/Sanderling/releases).
```

Comment:
yes sorry it's because i havechange thtat for my github and forget it when i have make my pull request